packages/expo-blur/src/BlurView.types.ts

```diff
@@ -1,6 +1,6 @@
 import { View } from 'react-native';
 
-export type Props = {
+export type BlurProps = {
```

Comment:
@esamelson, I and @lukmccall discussed what approach should be taken when facing already exported `type`/`interface` that name is generic like this one.
Changing it like @lukmccall did would definitely benefit in long run (no missimporting by IDE or programmer), but would cause a little breaking change (what if somebody has already written: `import { Props } from 'expo-blux'` ?) . Moreover I expect that documentation will have to be changed a little regarding previously exported types (if applicable - [expo-blur docs here](https://docs.expo.io/versions/latest/sdk/blur-view/#props) ).
I suggest proceeding with such breaking changes to catch up with react native typing conventions ([ScrollViewProps here for reference](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts#L6238) ).

Reply:
I'm not sure if we should add a prefix to all of our exported types.

In many cases, users will import them like this:
```
import * as X from 'expo-x';
```

And then use it like this:
```
const variable: X.type = {
...
}
```

Also, in most of our modules, we don't have prefixes. So, it'll require a lot of changes.
On the other hand, I think we should add a prefix to some of the types exported in this PR.
For example, to every prop types.

Reply:
I think the convention that we should use is `ComponentNameProps` - so `BlurViewProps` or `LinearGradientProps` - other well-built component libraries, eg: Shopify's [polaris-react](https://github.com/Shopify/polaris-react/blob/master/src/components/index.ts) uses the same convention.

If it's a breaking change that's not the end of the world, we can just document it and it should be easy enough for users to fix!

Reply:
@brentvatne, I agree that we should add a prefix to all props types. Cause, `Props` name is too generic. However, I not sure what we supposed to do with other types. For example, types like: `VoiceQuality`, `Voice` from `expo-speech` or `CapturedPicture`,  `PictureOptions`, `RecordingOptions` from 'expo-camera`. They should be prefixed or no?

Reply:
If we export types, they need to be names that mostly context-free and are likely to be globally unique. For example, `CameraRecordingOptions` is better than trying to be clever with `RecordingOptions` because it makes it clear that the type is related to the camera library and is unlikely to conflict with recording options types from other libraries (ex: `MicrophoneRecordingOptions`).

Most of our APIs have not undergone an audit by Eric and/or myself and are not examples to follow.

Reply:
yeah i think prefixing for all exports is very reasonable!